workflows/cumulus/cumulus.wdl

```diff
@@ -1,7 +1,7 @@
 version 1.0
 
-import "https://api.firecloud.org/ga4gh/v1/tools/cumulus:cumulus_tasks/versions/31/plain-WDL/descriptor" as tasks
-
+#import "https://api.firecloud.org/ga4gh/v1/tools/cumulus:cumulus_tasks/versions/31/plain-WDL/descriptor" as tasks
```

Comment:
You can remove this line, as we won't use Broad Method Registry URLs any more.

Reply:
removed the URL